report/src/main/webapp/assets/sass/_sidebar.scss

```diff
@@ -147,7 +147,7 @@
         color: $gray_dark;
 
         &:last-of-type {
-          border-bottom: solid 1px #313942;
```

Comment:
If this line is not needed let's remove it.
If someone wants it back we will search it in commit history.

Let's not have commented-out code.

Reply:
Lines are removed in a new commit.